### PR TITLE
Add filter in getting cleared stage id.

### PR DESCRIPTION
**Pull request by Unengine**
Filter hard mode stage ID in `TryGetLastStageId`.

### CODE REVIEW

File: Lib9c/GameConfig.cs [hunk 1]
```diff
@@ -40,6 +40,8 @@ public static class GameConfig
         public const int ArenaScoreDefault = 1000;
         public const int ArenaChallengeCountMax = 5;
 
+        public const int HardStageBeginId = 101;
```
Comment (boscohyun) [inline Lib9c/GameConfig.cs:43]: https://github.com/planetarium/lib9c/pull/227 PR contains [this line](https://github.com/hskim881028/lib9c/blob/4ab035a7f544804ae08c4a0729d0b3c82546d8b3/Lib9c/GameConfig.cs#L20)

Comment (Unengine) [reply Lib9c/GameConfig.cs:43]: I filtered with `World.Id`, but is it better to filter with `Stage.Id`?

**Review by ipdae — APPROVED** [review]

**Review by hskim881028 — APPROVED** [review]
Previously, if you had cleared any of the Stages in Mimirsbrunnr, you only get mimisbrunnr stage id. So you divided the method.

**Review by Unengine — COMMENTED** [review]
@boscohyun LGTM!

**Pull request merged**